src/Common/SignalHandlers.cpp

```diff
@@ -634,7 +634,7 @@ void HandledSignals::setupCommonDeadlySignalHandlers()
 {
     /// SIGTSTP is added for debugging purposes. To output a stack trace of any running thread at anytime.
     /// NOTE: that it is also used by clickhouse-test wrapper
-    addSignalHandler({SIGABRT, SIGSEGV, SIGILL, SIGBUS, SIGSYS, SIGFPE, SIGPIPE, SIGTSTP, SIGTRAP}, signalHandler, true);
+    addSignalHandler({SIGABRT, SIGSEGV, SIGILL, SIGBUS, SIGSYS, SIGFPE, SIGTSTP, SIGTRAP}, signalHandler, true);
```

Comment:
https://github.com/llvm/llvm-project/pull/98200

Also, why do we block it [here](https://github.com/ClickHouse/ClickHouse/blob/3272f1143c42a93058dad167adbbcf2f12058654/src/Daemon/BaseDaemon.cpp#L436), but then add a handler?